packages/edit-post/src/components/layout/index.js

```diff
@@ -400,6 +400,10 @@ function Layout( {
 				kind: 'postType',
 				name: 'wp_template',
 			} );
+			const canDiscoverTemplate = canUser( 'read', {
+				kind: 'root',
+				name: 'site',
+			} );
```

Comment:
Maybe we should move this check into the `getHomePage` selector so we don't have to remember this implementation detail.

Reply:
Good point I added the check inside the selector.